Source/Modules/main.cxx

```diff
@@ -105,6 +105,7 @@ static const char *usage2 = "\
      -I-             - Don't search the current directory\n\
      -I<dir>         - Look for SWIG files in directory <dir>\n\
      -ignoremissing  - Ignore missing include files\n\
+     -ignoreattrs    - Ignore C++11/C23 [[attributes]]\n\
```

Comment:
Making this a command line option seems unhelpful - it's something the user is likely to need to specify for a particular interface file (because it's something that is likely to be needed based on particular uses of attributes in third party headers they're trying to wrap), so it would be better to be able to specify it in the interface file (if it's a global setting, perhaps as a parameter on `%module`; if it's something that's useful to control in a more fine-grained way, perhaps as a `%feature`.  I can see one might theoretically want it on for one wrapped header but not another, but that may be a bit of a contrived example).

Reply:
I did mostly because the specifications for `[[attribute]]` are very vague and I am afraid that now that parsing is implemented and mandatory, there might be cases where the parser wouldn't expect an `[[attribute]]` and this header won't pass at all in SWIG.

Reply:
This is a parser option, it does not belong in the parsed text. Since an attribute cannot contain a `%feature` it can be made to work, but still, I don't think it belongs there.